Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/zip.test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+SCRIPT=$(realpath "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd "$SCRIPTPATH"
+
+./zip.sh
+unzip ./Package/IncidentProducer.zip -d IncidentProducer.output1
```

Comment:
Should we do the same for consumer as well?

Reply:
yes, my latest change do this.